src/Microsoft.DotNet.Build.Tasks/FilterRuntimesFromSupports.cs

```diff
@@ -4,18 +4,20 @@
 
 using System.Collections.Generic;
 using Newtonsoft.Json.Linq;
+using NuGet.Frameworks;
 using NuGet.RuntimeModel;
 
 namespace Microsoft.DotNet.Build.Tasks
 {
     public class TfmRidPair
     {
-        public TfmRidPair(string tfm, string rid)
+        public TfmRidPair(NuGetFramework tfm, string rid)
         {
             framework = tfm;
             runtimeIdentifier = rid;
         }
-        public string framework { get; set; }
+        //This should be the long form of the Nuget Framework moniker.
```

Comment:
I would have expected you to continue to use a string, but just make sure that you're passing in the long form.